### PR TITLE
add restricted settings help topic

### PR DESCRIPTION
**Pull request by gibbiemonster**
Adds a help topic for restricted settings.

### CODE REVIEW

File: supportMap.json [hunk 1]
```diff
@@ -363,7 +363,7 @@
       "donations"
     ],
     "keywords": [
-      ["donat"]
```
Comment (Willow-Systems) [inline supportMap.json]: This is correct. It's incomplete so it will match 'donate', 'donations', 'donating' etc.

File: supportMap.json [hunk 2]
```diff
@@ -392,6 +392,21 @@
...
+      "restricted"
+    ],
+    "keywords": [
+      ["restricted", "settings"],
+      ["restricted"]
+    ],
+    "title": "Restricted settings",
+    "lastupdated": "2023-01-03T13:40:19.402Z",
+    "description": "You run into an error enabling notifications on Android 13+.",
+    "fixurl": "https://help.rebble.io/android-restricted-settings",
```
Comment (Willow-Systems) [inline supportMap.json:407]: This looks good! There are some changes needed blocking the help centre PR, but once that's done and merged we can merge this too

**Review by Willow-Systems — CHANGES_REQUESTED** [review]

**Comment by gibbiemonster** [comment]
Okay @Willow-Systems - this is ready now!

**Comment by Willow-Systems** [comment]
It's live! Thanks for this, it's a useful addition :)

**Pull request merged**